packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -19,7 +19,7 @@ interface RuleMetaDataDocs {
   /**
    * The URL of the rule's docs
    */
-  url: string;
+  url?: string;
```

Comment:
@bradzacher would you be upset if I kept this optional and added a `?` to fix the test below?

```
tests/docs.test.ts:114:11 - error TS2532: Object is possibly 'undefined'.

114           rule.meta.docs?.url.endsWith(`rules/${ruleName}.md`),
              ~~~~~~~~~~~~~~~~~~~
```

An alternative would be to make `*WithDocs`/`*WithUrl` versions of the types to handle these being known internally...

Reply:
nah this should be fine.
it's actually the type you'd see in the wild anyways.